run-directory.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+set -e
+
+. venv/bin/activate
+
+python3 directory.py
```

Comment:
:+1:  

I'm confused by the filename, please name it by what it is/does

Reply:
By using `git mv directory.py <new-name.py>` , git will notice it's renamed  :+1: